lib/addons/neuron/index.ts

```diff
@@ -0,0 +1,31 @@
+import { KubernetesManifest } from "aws-cdk-lib/aws-eks";
+import { ClusterAddOn, ClusterInfo } from "../../spi";
+import { loadYaml, readYamlDocument } from "../../utils/yaml-utils";
+
+export class NeuronPluginAddOn implements ClusterAddOn {
+    deploy(clusterInfo: ClusterInfo): void {
+        const cluster = clusterInfo.cluster;
+
+        // Read in YAML docs
+        const rbac = readYamlDocument(__dirname + '/k8s-neuron-device-plugin-rbac.ytpl');
```

Comment:
Minor (can be deferred): suggestion - create a utility function `readMultiDocYaml` and wrap the `split().map(load)` logic in it.

Reply:
fixed the existing `loadMultiResourceExternalYaml` function to do this. Addressed.